precompile/contracts/feemanager/module.go

```diff
@@ -0,0 +1,60 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package feemanager
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/subnet-evm/precompile/config"
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var _ contract.Module = Module{}
+
+// ConfigKey is the key used in json config files to specify this precompile config.
+// must be unique across all precompiles.
+const ConfigKey = "feeManagerConfig"
+
+var ContractAddress = common.HexToAddress("0x0200000000000000000000000000000000000003")
+
+type Module struct{}
+
+func (Module) Key() string {
+	return ConfigKey
+}
+
+// Address returns the address of the fee manager.
```

Comment:
probably removing this comment makes the template gen. easier

Reply:
removed in #509